backend/src/nodes/properties/outputs/pytorch_outputs.py

```diff
@@ -21,6 +21,19 @@ def get_broadcast_data(self, value: PyTorchModel):
 
         if "SRVGG" in value.model_type:  # type: ignore
             size = [f"{value.num_feat}nf", f"{value.num_conv}nc"]
+        elif "SwinIR" in value.model_type:  # type: ignore
+            head_length = len(value.depths)  # type: ignore
+            if head_length <= 4:
+                size_tag = "small"
+            elif head_length < 9:
+                size_tag = "medium"
+            else:
+                size_tag = "large"
```

Comment:
This seems arbitrary. Could you explain this?

Reply:
https://github.com/JingyunLiang/SwinIR/releases/tag/v0.0

The S, M and L in the model names

Reply:
I guess it's fine then.